usr/sbin/genRegionServerCert

```diff
@@ -145,6 +145,11 @@ argparse.add_option(
     help='Add an additional hostname for which the certificate is valid',
     dest='san_dns'
 )
+argparse.add_option(
+    '-e', '--expire-date-years',
+    help='Set the number of years for the expiration date of the cert',
+    dest='exp_date_years'
```

Comment:
I think we should have a default of 10 years such that a user does not have to specify the years if not desired.